sdk/python/kfp/dsl/dsl_utils.py

```diff
@@ -97,6 +98,8 @@ def _output_artifact_path_placeholder(output_key: str) -> str:
 def _output_parameter_path_placeholder(output_key: str) -> str:
   return "{{{{$.outputs.parameters['{}'].output_file}}}}".format(output_key)
 
+def _intput_executor_placeholder() -> str:
```

Comment:
let's name this `_executor_input_placeholder` instead.

Reply:
Done :)